src/style.css

```
@@ -1 +1,79 @@
 /* Styles go here */
+@import url(http://fonts.googleapis.com/css?family=Roboto:500);
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.navList {
+  display: flex;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+}
+
+/* .navList:not(:last-child) {
+  margin-right: 20px;
+} */
```

Uncomment it to fix you tests, but remake it for navItem
```suggestion
.navItem:not(:last-child) {
  margin-right: 20px;
}
```